_data/logiciels_libres-open_source_software/mediawiki.yml

```diff
@@ -2,7 +2,7 @@
 schemaVersion: '1.0'
 description:
   en: Wiki package written in PHP, originally for use on Wikipedia
-  fr: Ensemble Wiki développé à l’origine pour Wikipédia
```

Comment:
Changed apostrophes for single quote as it was throwing an error when updating with jsyaml format.